### PR TITLE
[Term Entry] JavaScript Storage .getItem(key) - close #2694

### LINKED ISSUE #2694: [Term Entry] JavaScript Storage .getItem(key)
### Reason/inspiration (optional)

We would like a new term entry for JavaScript [Storage](https://www.codecademy.com/resources/docs/javascript/storage) about the .getItem() method. The entry should go in a new file in docs/content/javascript/concepts/storage/terms/getItem.md

The entry should
    - Have an introductory paragraph about what the method is and does.
    - A syntax section with a pseudo code block with an explanation of it.
    - An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by SimonaPiz**
### Description
close #2694

This was my first entry for Codecademy docs

- I created a new folder 'storage/terms/getItem' in 'content/javascript/concepts/'
- Inside it I created new file 'getItem.md' with the required content
  - [x] Title
  - [x] Description
  - [x] Subjects
  - [x] Tags
  - [x] CatalogContent
  - [x] Syntax
  - [x] Codebyte Example
- I added a new `Storage` tag  in '/documentation/tags.md'

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - 'paths/back-end-engineer-career-path'
+---
+
+The **.getItem()** method takes the key of the data item as an argument and returns the value of the data if it is in the `Storage` object, otherwise it returns `null`.
+
+The `Storage` Object, which can be either a `localStorage` object or a `sessionStorage` object.
+
+## Syntax
+
+```javascript
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:27]:
```suggestion
```pseudo
```
use pseudo tags for syntax

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+localStorage.getItem(keyName)
+```
+or
+```javascript
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:31]:
```suggestion
or

```pseudo
```

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Title: '.getItem(key)'
+Description: 'Returns the value of the key of the data item passed as an argument or null if it does not exist, from the storage object'
+Subjects: 
+  - 'Web Development'
+  - 'Computer Science'
+Tags: 
+  - 'APIs'
+  - 'Data'
+  - 'Methods'
+  - 'Objects'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:11]:
```suggestion
```

Comment (caupolicandiaz) [reply content/javascript/concepts/storage/terms/getItem/getItem.md:11]: we limit tags to 5 items

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  - 'Storage'
+  - 'Values'
+CatalogContent: 
+  - 'user-authentication-authorization-express/modules/session-authentication-express/cheatsheet'
+  - 'user-authentication-authorization-express'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:16]:
```suggestion
```

Comment (caupolicandiaz) [reply content/javascript/concepts/storage/terms/getItem/getItem.md:16]: catalog content should only contain two items

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  - 'paths/securing-express-applications'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:17]:
```suggestion
  - 'introduction-to-javascript'
```

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  - 'paths/back-end-engineer-career-path'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:18]:
```suggestion
  - 'paths/front-end-engineer-career-path'
```

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+The **.getItem()** method takes the key of the data item as an argument and returns the value of the data if it is in the `Storage` object, otherwise it returns `null`.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:21]:
```suggestion
The **`.getItem()`** method takes a key as an argument and returns the associated value. If the key is not present in the `Storage` object, it returns `null`.
```

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
+---
+Title: '.getItem(key)'
+Description: 'Returns the value of the key of the data item passed as an argument or null if it does not exist, from the storage object'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:3]:
```suggestion
Description: 'Returns the value of the key on the given data item.'
```

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+or
+```pseudo
+sessionStorage.getItem(keyName)
+```
+
+The `keyName` parameter is required and it's a string containing the name of the key you want to retrieve the value of.
+
+> You could also use `Storage.length` to test whether the storage object is empty or not.
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:39]:
```suggestion
## Example
```

Comment (caupolicandiaz) [reply content/javascript/concepts/storage/terms/getItem/getItem.md:39]: this function is not currently supported by Codecademy's compiler. Please reframe this example as a "standard" example with an output in a ```shell block

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+localStorage.getItem(keyName)
+```
+or
+```pseudo
+sessionStorage.getItem(keyName)
+```
+
+The `keyName` parameter is required and it's a string containing the name of the key you want to retrieve the value of.
+
+> You could also use `Storage.length` to test whether the storage object is empty or not.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:37]:
```suggestion
> **Note:** The `Storage.length` property can be used to test whether the storage object is empty or not.
```

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+  - 'paths/back-end-engineer-career-path'
+---
+
+The **.getItem()** method takes the key of the data item as an argument and returns the value of the data if it is in the `Storage` object, otherwise it returns `null`.
+
+The `Storage` Object, which can be either a `localStorage` object or a `sessionStorage` object.
+
+## Syntax
+
+```pseudo
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:27]: the two formats can be consolidated into one block

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@SimonaPiz I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by SimonaPiz** [comment]
@caupolicandiaz 
Thanks for the review and recommendations. I have made the requested changes.

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+The `keyName` parameter is required and it's a string containing the name of the key you want to retrieve the value of.
+
+> **Note:** The `Storage.length` property can be used to test whether the storage object is empty or not.
+
+## Example
+
+Get the value of the specified local storage item:
+
+```codebyte/javascript
```
Comment (caupolicandiaz) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:38]: There are still changes missing here. The block is still set as a codebyte and the code should be revised to be a more complete example that could be conceivably used as is, something like:
localStorage.setItem("Seuss", "Green Eggs and Ham")
console.log(localStorage.getItem("Seuss")

And the output should be in a separate ```shell block

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@SimonaPiz I've added additional comments. Please review and revise.

**Review by caupolicandiaz — APPROVED** [review]
@SimonaPiz I've made some small edits to your revisions, please review for future reference. I'm passing this on for a second review.

File: content/javascript/concepts/storage/terms/getItem/getItem.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+
+## Syntax
+
+```pseudo
+localStorage.getItem(keyName)
+
+sessionStorage.getItem(keyName)
+```
+
+The `keyName` parameter is required and it's a string containing the name of the key you want to retrieve the value of.
```
Comment (SSwiniarski) [inline content/javascript/concepts/storage/terms/getItem/getItem.md:30]:
```suggestion
The `keyName` parameter is required and it's a string containing the name of the key to retrieve the value of.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@SimonaPiz I did a review and had one comment. (We avoid first- and second-person pronouns in Docs.)

**Review by SSwiniarski — APPROVED** [review]
@SimonaPiz Looks good to merge once we have a storage concept merged.

**Pull request merged**